src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp

```diff
@@ -241,23 +241,26 @@ class KerrSchild : public AnalyticSolution<3_st>,
 
   explicit KerrSchild(CkMigrateMessage* /*unused*/) {}
 
+  template <typename DataType, typename Frame = Frame::Inertial>
+  using tags = tmpl::flatten<tmpl::list<
+      AnalyticSolution<3_st>::tags<DataType, Frame>,
+      gr::Tags::DerivDetSpatialMetric<3, Frame, DataType>,
```

Comment:
I would prefer adding all 3 tags into `AnalyticSolution::tags` so that all solutions have to supply them. It can be a bit annoying having different solutions missing some tags. Given that the computations are relatively straight forward, it shouldn't be too bad to add it to KS, Spherical KS, and Minkowski. Is that okay?

Reply:
I have added these to `KS`, `spherical KS` and `Minkowski`. When I tried to add these to `AnalyticalSolutions::tags`, the other solutions `HarmonicSchwarzschild` and `GaugeWave` broke since they do not have these tags either. I am not going to add e.g. `DerivDetSpatialMetric` to these which I would have to calculate first.

I think the tags in `AnalyticSolutions` should really be the minimal tags that could be needed so a developer can simply add a new solution without needing to worry about adding a large number of functions which may never be used.

Reply:
Okay, sounds good. Thanks for giving that a try!